server/common/utils/index.js

```diff
@@ -107,8 +101,19 @@ const responseWithCohort = (cohort, userId) => {
   };
 };
 
-const checkIfOwner = (ownerIds, userId) =>
-  ownerIds.indexOf(userId.toString()) > -1;
+const checkIfArrayContainsUserId = (idsArray, userId) => {
```

Comment:
do we need this method - can't we just use `includes` from `lodash`? Do we need to convert everything to strings?

Reply:
I've checked this solution with includes, earlier and it doesn't work cause it's doesn't recognize ObjectId format and cannot compare to userId 

Also checking on stackoveflow, some other guy had the same problem, and one of the answers was "convert objectId to string"